docs/rfc/README.md

```diff
@@ -35,6 +35,4 @@ sections.
 
 ## Table of Contents
 
-<!-- - [RFC-NNN: Title](./rfc-NNN-title.md) -->
```

Comment:
This should stay, this is a template to help people to format properly the RFCs in the ToC.